src/linear_algebra/free_module_pid.lean

```diff
@@ -1,17 +1,19 @@
 /-
-Copyright (c) 2020 Anne Baanen. All rights reserved.
```

Comment:
It looks like something has gone wrong here, and you replaced this file with `free_module`? I assume just reverting all changes to this file from the PR will fix things?

Reply:
I must have messed up something with GIT, it should be fixed now.